tgstation.dme

```diff
@@ -221,6 +221,7 @@
 #include "code\datums\helper_datums\teleport.dm"
 #include "code\datums\helper_datums\topic_input.dm"
 #include "code\datums\martial\krav_maga.dm"
+#include "code\datums\martial\SNAP_INTO_A_SLIM_JIM.dm"

```

Comment:
Either way rename the file to something more descriptive about what you're going to find. Not everyone knows every meme ever